docs/references/kontena-yml.md

```diff
@@ -15,6 +15,124 @@ You can use environment variables in configuration values with a Bash-like ${VAR
 ## Service configuration reference
 > **Note:** Kontena supports both Docker Compose file versions respectively. Volumes and networks introduced in version 2 are not supported by Kontena yet. See more details about versioning on [Docker Compose documentation](https://docs.docker.com/compose/compose-file/#versioning)
```

Comment:
Maybe we don't need that "not supported" section because we list all supported keys.. everything else is not supported?